### PR TITLE
fix syntax to remove html tags

### LINKED ISSUE #279: HTML tags showing
This is a screenshot from a community member. I'm not sure details to reproduce the bug (eg. browser, OS, etc)

image (80)

### PR DESCRIPTION
**Pull request by techtolentino**
## Changes

Updated syntax in formula to remove `<em>` tags
Closes #279

## Screenshots
Screen Shot 2022-02-22 at 11 19 21 AM

### CODE REVIEW

File: notebooks/quantum-machine-learning/qgan.ipynb [hunk 1]
```diff
@@ -22713,7 +22713,7 @@
  "metadata": {
   "celltoolbar": "Edit Metadata",
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
```
Comment (Tansito) [inline notebooks/quantum-machine-learning/qgan.ipynb:22716]: Can we remove this changes from the PR?

File: notebooks/quantum-machine-learning/qgan.ipynb [hunk 2]
```diff
@@ -22727,7 +22727,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.8"
+   "version": "3.9.1"
```
Comment (Tansito) [inline notebooks/quantum-machine-learning/qgan.ipynb:22730]: Same here, can we remove it?

**Review by Tansito — CHANGES_REQUESTED** [review]

**Review by eddybrando — APPROVED** [review]
Perfect! Thank you very much, @Tansito! 🙂

**Comment by techtolentino** [comment]
Thank you, @Tansito 

(not sure why those always update whenever I make a change in a notebook)

**Comment by Tansito** [comment]
It gets the environment configuration from the last run but doesn't affect anything, no problem.

**Pull request merged**